spec/tides.spec.json

```diff
@@ -0,0 +1,84 @@
+{
+  "profile": "tides-data-package",
+  "profile_version": 0,
+  "name": "tides-dataset",
+  "resources": [
+    {
+      "name": "vehicle_locations",
+      "description": "Timestamped vehicle locations and speeds.",
+      "path": "vehicle_locations.csv",
+      "schema": "vehicle_locations.schema.json",
+      "required": false
+    },
+    {
+      "name": "passenger_events",
+      "description": "Timestamped passenger-related events, including boardings and alightings.",
+      "path": "passenger_events.csv",
+      "schema": "passenger_events.schema.json",
+      "required": false
+    },
+    {
+      "name": "fare_transactions",
+      "description": "Timestamped fare transaction, associated with devices.",
+      "path": "fare_transactions.csv",
+      "schema": "fare_transactions.schema.json",
+      "required": false
+    },
+    {
+      "name": "stop_visits",
+      "description": "	Summarized boarding, alighting, arrival, departure, and other events (kneel engaged, ramp deployed, etc.) by trip and stop for each service date.",
+      "path": "Stop_visits.csv",
+      "schema": "stop_visits.schema.json",
+      "required": true
+    },
+    {
+      "name": "trips_performed",
+      "description": "Trips performed for each service date.",
+      "path": "trips_performed.csv",
+      "schema": "trips_performed.schema.json",
+      "required": true
```

Comment:
Are any of the files truly required? It seems a little odd to have vehicles, trips_performed and stop_visits required. But maybe that’s correct

Reply:
this is an interesting issue - b/c files (and fields) are required for certain "features" but not necc to comply with the spec itself...leaving me to think that we would want to make the spec as defined here as neutral and then if/when we define requirements for features - define more strict requirements within that.

Reply:
I like the idea of defining "use cases" for the spec that document the files and fields required for different processes and purposes.  This would allow someone to crosswalk between a given dataset, which documents the files and field included, and a given use case, to know whether the dataset supports the use case.  I think this would require not just the files and fields, but also some measure of the data quality (completeness, accuracy, precision, etc.) for the data that is present.

Reply:
Agreed. Can we then resolve this for now by:

1. @j-meelah updating to make these not required (unless they truly are for ALL use cases)
2. Opening an issue to define use-cases/features per @botanize 's comments

Reply:
1. Has been completed.